omero/sysadmins/in-place-import.rst

```diff
@@ -349,10 +349,10 @@ placed in the `lib/clients` directory, you can invoke it using:
 Related advanced options
 ------------------------
 
-In addition to the :option:`--transfer` option in 5.0.0, a number of other
+In addition to the ``--transfer`` option in 5.0.0, a number of other
```

Comment:
Not related to the PR, but I wonder if the 5.0.0 and similar references could be removed everywhere.  This has been standard functionality in all versions for years now, so the version doesn't add much value.